docs/docs/for-developers/building.md

```diff
@@ -95,11 +95,17 @@ This will create a `_release` folder at the root with the application bundle ins
 
 ### OSX
 
-Once you have a release build created, you can create an `oni2` symlink to point to your development environment.
+Once you have a release build created, you can install Oni2 on `/Applications` folder.
 
 Run the following from the `oni2` directory:
-- `./scripts/osx/create-symlink.sh`
 
+- `cp -R _release/Onivim2.app /Applications`
```

Comment:
Don't you typically install user apps in the user's `Applications` folder? Seems like this might require `sudo` as well.